### PR TITLE
Add market making fees since

### PR DESCRIPTION
**Pull request by dospore**
Add Market making fees

### CODE REVIEW

File: src/Api/index.js [hunk 1]
```diff
@@ -74,6 +78,44 @@ export function useFees(chainId) {
...
+      margin
+    }
+  }`);
+
+  useEffect(() => {
+    if (!from) {
+      return
+    }
+    getMycGraphClient(chainId).query({ query }).then((res) => {
+      console.log(res);
```
Comment (rogue-hacker) [inline src/Api/index.js]: Is this still required

File: src/views/Dashboard/DashboardV2.js [hunk 2]
```diff
@@ -146,6 +146,12 @@ export default function DashboardV2() {
   const allFees = useFees(chainId);
 
   const feeHistory = getFeeHistory(chainId);
+  const currentWeeksMMFees = useMarketMakingFeesSince(chainId, feeHistory[0]?.to)
+  console.log(currentWeeksMMFees)
```
Comment (rogue-hacker) [inline src/views/Dashboard/DashboardV2.js]: Another log

**Review by rogue-hacker — CHANGES_REQUESTED** [review]

File: src/Api/index.js (hunk 1 above)
Comment (dospore) [reply src/Api/index.js]: ty

File: src/views/Dashboard/DashboardV2.js (hunk 2 above)
Comment (dospore) [reply src/views/Dashboard/DashboardV2.js]: ty

File: src/views/Dashboard/DashboardV2.js [hunk 3]
```diff
@@ -475,7 +480,20 @@ export default function DashboardV2() {
...
+                        handle={`$${formatAmount(totalCurrentFees, USD_DECIMALS, 2, true)}`}
+                        renderContent={() => (
+                          <>
+                            Distributed Fees: ${formatAmount(currentFeesUsd, USD_DECIMALS, 2, true)}
+                            <br />
+                            Spread Capture: ${formatAmount(currentWeeksMMFees, USD_DECIMALS, 2, true)}
+                          </>
+                        )}
+                      />
+                    </div>
```
Comment (rogue-hacker) [inline src/views/Dashboard/DashboardV2.js:496]: The numbers here don't quite add up in the UI, is there a small disparity on purpose? 

image

**Review by rogue-hacker — APPROVED** [review]

File: src/views/Dashboard/DashboardV2.js (hunk 3 above)
Comment (dospore) [reply src/views/Dashboard/DashboardV2.js:496]: lol https://github.com/mycelium-ethereum/swaps-client/pull/115/files#diff-bf29c66c1d51657e91e65f4fd1b1cb1ab36f14b5a3df3dae557b9c0cd47a3f25R152

**Review by rogue-hacker — APPROVED** [review]

**Pull request merged**